src/Core/Update.php

```diff
@@ -286,7 +286,7 @@ private static function updateConfigEntry(IConfigCache $configCache, ConfigFileS
 		$savedConfig = DBA::selectFirst('config', ['v'], ['cat' => $cat, 'k' => $key]);
 
 		if (!DBA::isResult($savedConfig)) {
-			return false;
+			$savedConfig = null;
```

Comment:
`$savedConfig` is expected to be an array after this.

Reply:
Ah, I was lazy and thought I can reuse the variable ;-).